contract/test/test-build-proposal.js

```diff
@@ -4,10 +4,31 @@ import { execSync } from 'node:child_process';
 import { promises as fs } from 'node:fs';
 import { makeCompressFile } from './utils.js';
 
-test.before(t => (t.context.compressFile = makeCompressFile(fs.readFile)));
+/**
+ * @import {ExecutionContext} from 'ava';
+ */
+/**
+ * @typedef {object} TestContext
+ * @property {function(any): Promise<Buffer>} compressFile
+ */
+
+/**
+ * @param {ExecutionContext<TestContext>} t
+ */
```

Comment:
please just type `test`. the rest will fall out of that.
https://github.com/Agoric/agoric-sdk/blob/136f1caa919428684a88f2213d0cd9b3185f8a33/packages/boot/test/configs.test.js#L20-L24

```js
const test =
  /** @type {import('ava').TestFn<Awaited<ReturnType<typeof makeTestContext>>>}} */ (
    anyTest
  );
```